doc/cookbook/source/examples/classifier/LDA.rst

```diff
@@ -0,0 +1,52 @@
+=============================
+Linear Discriminant Analysis
+=============================
+
+LDA learns a linear classifier that attempts to find a projection matrix that maximally discriminates the provided classes. The learned linear classification rule is optimal under the assumption that both classes a gaussian distributed with equal co-variance. To find a linear separation :math:`{\bf w}` in training, the in-between class variance is maximized and the within class variance is minimized.
```

Comment:
LDA learns a classifier via finding a projection matrix.

The classifier doesn't attempt to find a matrix